helm/prometheus-rules/templates/alerting-rules/fairness.rules.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: fairness.rules
+  namespace: {{ .Values.namespace  }}
+spec:
+  groups:
+  - name: fairness
+    rules:
+    - alert: FlowcontrolRejectedRequests
+      annotations:
+        description: '{{`Cluster {{ $labels.cluster_id }} .`}}'
```

Comment:
Can we add more context here? Like `Cluster $clusterid k8s api is rejecting requests due to fairness subsystem` or whatever?

Reply:
sure fuck me